### PR TITLE
Update Spyder 3 icon theme: vertical alignment of maximize/unmaximize

### PR DESCRIPTION
**Pull request by SylvainCorlay**
Fixing the vertical alignment issues mentioned by @goanpeca in #2490.

### CODE REVIEW

**Comment by goanpeca** [comment]
Image :) ?

**Comment by SylvainCorlay** [comment]
Before:
before

After:
after

**Comment by goanpeca** [comment]
Lovely

**Comment by goanpeca** [comment]
@SylvainCorlay maybe you should make an entry in the wiki explaining the process of making a new icon from scratch using the tools you have used so far

**Comment by bmer** [comment]
Is there a wiki that documents how to extend Spyder?

**Comment by goanpeca** [comment]
You mean with plugins? Yes and no. The material is partly written but is not public yet as the plugin interface is still under testing.

**Comment by goanpeca** [comment]
@SylvainCorlay, with the risk of sounding annoying, I must insist that we use filled icons to have a more homogeneous looking UI.

http://fortawesome.github.io/Font-Awesome/icon/file/

Also, I have a design for a filled in save as icon, if you can point me in the right direction (procedure and tools used to make the custom fonts, it would be great!, more people will likely benefit from this)

Cheers

**Comment by SylvainCorlay** [comment]
For the icon design, I will definitely write a tutorial in the wiki, hopefully before Scipy.

**Comment by SylvainCorlay** [comment]
Regarding other icon design, I am not addressing all the comments of #2490 here. This is only for the alignment issues.

**Comment by goanpeca** [comment]
Great ;-)

File: spyderlib/utils/icon_manager.py [hunk 1]
```diff
@@ -40,7 +40,7 @@
     'arrow-continue':          [('spyder.continue',), {'color': '#3775a9'}],
     'arrow-step-in':           [('spyder.step-into',), {'color': '#3775a9'}],
     'arrow-step-out':          [('spyder.step-out',), {'color': '#3775a9'}],
-    'stop_debug':              [('spyder.stop',), {'color': '#3775a9'}],
+    'stop_debug':              [('fa.stop',), {'color': '#3775a9'}],
```
Comment (ccordoba12) [inline spyderlib/utils/icon_manager.py]: Why this change?

Comment (SylvainCorlay) [inline spyderlib/utils/icon_manager.py]: When I first made the spyder font, I recreated a lot of icons duplicating some that were included in font-awesome.
I eventually removed most of them.

Here I stop using spyder.stop to the benefit of fa.stop and I removed 'stop' from the spyder font.

**Comment by SylvainCorlay** [comment]
I opened a separate PR for the new run-cell icons.

File: spyderlib/utils/icon_manager.py (hunk 1 above)
Comment (ccordoba12) [inline spyderlib/utils/icon_manager.py]: So we're going to have a mix of png and fa icons in the Spyder 2 theme?

Comment (SylvainCorlay) [inline spyderlib/utils/icon_manager.py]: The Spyder 2 theme is the same as before.

The Spyder 3 theme is a mix of font-awesome and the custom Spyder font which has some icons wh8ch are not in font-awesome.

**Comment by SylvainCorlay** [comment]
@ccordoba12 are you in Scipy next week?

File: spyderlib/utils/icon_manager.py (hunk 1 above)
Comment (SylvainCorlay) [inline spyderlib/utils/icon_manager.py]: So we  two ttf files: font-awesome.ttf and spyder.ttf, all with qtawesome.

**Comment by ccordoba12** [comment]
Ok, thanks for the clarification. Merging then :-)

**Comment by ccordoba12** [comment]
@SylvainCorlay, yeah! I'm already at Austin to visit Continuum office :-) And you?

**Comment by SylvainCorlay** [comment]
I am flying tonight and will be at the conference the entire week until Saturday.

**Comment by ccordoba12** [comment]
Great!! @blink1073 is going to be here too, so Spyder representation is really big this year! ;-)

Please write me an email if you have time to grab some beers this weekend.

**Comment by goanpeca** [comment]
:'-(

**Comment by ccordoba12** [comment]
Maybe next year @goanpeca ;-)

**Comment by SylvainCorlay** [comment]
Totally, see you guys soon. Cheers. S.
